### DISCUSSION TITLE
Roles of members within the repo

### DISCUSSION THREAD

**Discussion by boeckera** [dynamics-of-stellar-systems/dynamite#375] [General]
@prashjet 
I cannot add people to teams. Iris and Anja need to be added to the webmasters team.
I think it might be because you, Laura and Glenn are owners of the repo and/or within the teams webmasters and maintainers you are the only one with the maintainer role, all others are only members.
Should we maybe discuss and fix the roles of people?

**Comment by prashjet** [comment]
Hey hey. @boeckera I've made you an owner of our organisation. Can discuss any other reorganisation soon :)
